src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/authorization/ServiceProviderAccessScopeMapper.kt

```diff
@@ -57,7 +59,19 @@ class ServiceProviderAccessScopeMapper(
     return ServiceProviderAccessScope(
       serviceProviders = workingScope.providers,
       contracts = workingScope.contracts,
-    )
+    ).also {
+      telemetryClient.trackEvent(
+        "InterventionsAuthorizedProvider",
+        mapOf(
+          "userId" to user.id,
+          "userName" to user.userName,
+          "userAuthSource" to user.authSource,
+          "contracts" to it.contracts.joinToString(",") { c -> c.contractReference },
+          "providers" to it.serviceProviders.joinToString(",") { p -> p.id },
```

Comment:
Maybe should order to ensure equality when contracts/providers are the same but difference order

Reply:
Good shout. Perhaps I should do that on the query level 🤔

Reply:
Done in 8e7ee35ee5bf7bde57929cb73456165390294273